File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/CQS/TransferFromFunction.vb
Imports System
Imports System.Threading.Tasks
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.Contracts.CQS
Imports Nethereum.ABI.FunctionEncoding.Attributes
Imports StandardToken.MyContractName.DTOs
Namespace StandardToken.MyContractName.CQS

    <[Function]("transferFrom", "bool")>
    Public Class TransferFromFunction
        Inherits ContractMessage
    
        <[Parameter]("address", "_from", 1)>
        Public Property From As String
        <[Parameter]("address", "_to", 2)>
        Public Property To As String
        <[Parameter]("uint256", "_value", 3)>
        Public Property Value As BigInteger
    
    End Class


End Namespace

File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/CQS/TransferFromFunction.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.Contracts.CQS;
using Nethereum.ABI.FunctionEncoding.Attributes;
using StandardToken.MyContractName.DTOs;
namespace StandardToken.MyContractName.CQS
{
    [Function("transferFrom", "bool")]
    public class TransferFromFunction:ContractMessage
    {
        [Parameter("address", "_from", 1)]
        public string From {get; set;}
        [Parameter("address", "_to", 2)]
        public string To {get; set;}
        [Parameter("uint256", "_value", 3)]
        public BigInteger Value {get; set;}
    }
}
